src/main/tls.c

```diff
@@ -3294,6 +3301,10 @@ SSL_CTX *tls_init_ctx(fr_tls_server_conf_t *conf, int client)
 
 	SSL_CTX_set_options(ctx, ctx_options);
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && OPENSSL_VERSION_NUMBER < 0x10101000L
+	SSL_CTX_set_max_early_data(ctx, 0);
```

Comment:
I'd add a comment "TLS 1.3 introduces the concept of early data (also known as zero round trip data or 0-RTT data). Early data allows a client to send data to a server in the first round trip of a connection, without waiting for the TLS handshake to complete if the client has spoken to the same server recently.  This doesn't work for EAP, so we disable early data"

Reply:
Resolved by https://github.com/FreeRADIUS/freeradius-server/pull/3516/commits/3abc3162cebb9df99eb579cd20fbe296283b1718